client/components/live-chat/index.jsx

```diff
@@ -261,6 +263,16 @@ const LiveChat = React.createClass( {
 		this.scrollToBottom();
 	},
 
+	onFocus() {
+		if ( viewport.isMobile() ) {
+			/* User tapped textfield on a phone. This shows the keyboard. Unless we scroll to the bottom, the chatbox will be invisible */
+			console.log( "Scroll to the bottom" );
```

Comment:
Instead of using `console.log` directly use `debug`:

```js
debug( 'Scroll to bottom' ); 
```
In the browser console you can setup a debug filter so it only shows the messages you care about:

```js
localStorage.debug='calypso:live-chat:*'
```